bundles/org.openhab.binding.elroconnects/src/main/java/org/openhab/binding/elroconnects/internal/handler/ElroConnectsDeviceConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.elroconnects.internal.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link ElroConnectsDeviceConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Mark Herwege - Initial contribution
+ */
+@NonNullByDefault
+public class ElroConnectsDeviceConfiguration {
+
+    public @Nullable Integer deviceId;
```

Comment:
```suggestion
    public Integer deviceId=0;
```

If its done like this and marked as required in the XML it saves it from needing to be marked as Nullable. Less code.